frontend/src/container/Publications.jsx

```diff
@@ -182,6 +188,35 @@ export class Publications extends React.Component {
     }
   }
 
+  handleChangeNewPublicationTitle = e => {
+    this.setState({ newPublicationTitle: e.target.value })
+  }
+
+  handleTogglePublicationTitlePopup = (publication, publicationAsFileList, commentReset) => {
+    const { state } = this
+
+    this.handleChangeNewPublicationTitle({ target: { value: '' } })
```

Comment:
be more specific about what is the parameter. Options are to declare a fake event or to update handleChangeNewPublicationTitle to receive the value and not the event (and update its usage accordingly)

Reply:
done